docs/modules/operation/pages/quick-start/service-assurance.adoc

```diff
@@ -0,0 +1,24 @@
+
+[[service-avaiability]]
+= Step 7: Determine Service Availability
+
+By default, {page-component-title} monitors a number of services including ICMP, DNS, Elasticsearch, SMTP, FTP, SNMP, HTTP(S), SSH, Postgres, IMAP, and POP3.
```

Comment:
Should we perhaps maintain a list of services that are monitored elsewhere? I don't know that having such an extensive list here is necessary.